_national/pycon-somalia.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: PyCon Somalia
+flag: so
+location: Somalia
+website: http://pycon.org.so/
```

Comment:
You may want to change this to https instead of http.

Reply:
> You may want to change this to https instead of http.

Thank you sir, i chaneged